### DISCUSSION TITLE
Analysis Setup Questions - Multiple Timepoint

### DISCUSSION THREAD

**Discussion by jstout211** [jonescompneurolab/hnn-core#933] [Q&A] [answered]
Hi All, 

I am just getting started with HNN, so maybe this is listed somewhere and I haven't seen it.  We have a few studies from patients that are undergoing an intervention.  So we have pre/post intervention datasets, and the intervention affects the morphology of the sensory responses.  From what I have seen with the tutorials, the data loaded into the HNN is generally group data.  In our datasets - although the group responses to stimulation show differences, the change is most apparent in within subject analysis.

Questions:
1) Is it viable to do the modelling on a single subject? 
2) What is the best way to compare pre/post intervention.  Do we optimize the parameters to the first time point and then run the simulation on the post-intervention data and see how the populations fire differently?   Or do we optimize on the pre-intervention and then optimize on the post intervention and just compare the simulation parameters somehow?
3) Same question, but with cross-sectional data.  If we have a group of healthy controls and patients - do we optimize on the HCs and then run the simulation on the patient data, or optimize on both? 

Thanks,
Jeff Stout

**Comment by jasmainak** [comment, marked as answer]
@jstout211 how I would approach is the following:

1) Create a base model that creates your waveform of interest (e.g., gamma, evoked response etc)
2) Once you have the base model, develop a hypothesis based on your intervention. What parameters of the model would you modify to obtain the change in signal morphology?
3) Sweep the parameter from the base model, say from 0.2 times the base parameter to 2 times the base parameter to understand how it affects the signal morphology
4) Tweak base model based on insight thus obtained and go back to step 2. Rinse and repeat.

Personally, I find parameter optimization pre-maturely leads to loss of insight. Once you understand the behavior of the model, you will be able to decide what's the best approach for within-subject vs between subject.

Tagging @rythorpe @ntolley @stephanie-r-jones for their take on the issue
